src/TcpServer.cpp

```diff
@@ -141,23 +146,16 @@ void TcpServer::run() {
             << inet_ntoa(this->socketAddress_.sin_addr)
             << " PORT: " << ntohs(this->socketAddress_.sin_port) << " ***\n\n";
   while (g_signaled == 0) {
-    if (PRINT) {
-      std::cout << "Sockets open: " << this->numfds_ << std::endl;
-    }
-    if (poll(this->pollfds_, this->numfds_ + 1, 100) == -1) {
+    if (poll(this->pollfds_, this->numfds_, 100) == -1) {
       perror("poll");
-      std::cout << this->numfds_ << std::endl;
       exitWithError("Poll failed");
     }
     checkSocketTimeout();
-    log("====== Waiting for a new connection ======\n\n\n");
-    checkUnfinished(this->sockets_);
+    // log("====== Waiting for a new connection ======\n\n\n");
```

Comment:
this might also be deleted, right?

Reply:
still a debug message that sprouted from my messy mind I'm sorry